.idea/checkstyle-idea.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="CheckStyle-IDEA">
+    <option name="configuration">
```

Comment:
@rishabhdeepsingh Don't send this file in pull request. It can be changed according to the configuration which is different for everyperson.

Reply:
ok